server/src/optima/project.py

```diff
@@ -229,6 +243,42 @@ def getProjectInformation():
         }
     return jsonify(response_data)
 
+@project.route('/list/all')
+@login_required
+@verify_admin_request
+def getProjectListAll():
+    """
+    Returns the list of existing projects from db.
+
+    Returns:
+        A jsonified list of project dictionaries if the user is logged in.
+        In case of an anonymous user an empty list will be returned.
```

Comment:
Not a big issue, but I noticed a lot of APIs/Apps usually return a 404 if a user tries to access something he/she should not be able to. This prevents bots from exploring endpoints & by this gathering data.